association_backend/.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
```

Comment:
is this a symbolic link to the .isort.cfg that is in the root?

Reply:
Now yes :simple_smile: